provisioning/roles/ewallet/templates/etc/profile.d/environment.sh.j2

```diff
@@ -1 +1,5 @@
-export LD_LIBRARY_PATH="/usr/local/lib:${LD_LIBRARY_PATH}"
\ No newline at end of file
+export LD_LIBRARY_PATH="/usr/local/lib:${LD_LIBRARY_PATH}"
+
+# These postgres users were created in /provisioning/roles/postgres/templates/srv/postgres/initdb/02_init_users.sql.j2
+export DATABASE_URL="postgres://ewallet:ewallet_weak_password@localhost/ewallet_dev"
```

Comment:
Just remembered… I actually avoid doing this because running tests will cause the app to use dev DB.

Reply:
Do we need this for test configs then?

```ex
  # ewallet_db/config/test.exs
  url: System.get_env("DATABASE_URL") || "postgres://localhost/ewallet_test"
```

Shall we just hardcode it to the latter string, or rename env to TEST_DATABASE_URL?

Reply:
I thought about having `TEST_DATABASE_URL` and fallback to `DATABASE_URL`… but then I'll have to use MFA tuple in releases to handle this env. 🤨 Maybe just change test to use `TEST_DATABASE_URL` I guess.

Reply:
Other idea is to not have separate users for `ewallet` and `ledgerdb` but make a `vagrant` as non-superuser and allow it to connect without password (in `pg_hba.conf`). This way we can use the default `DATABASE_URL`. Thoughts?

Reply:
> Other idea is to not have separate users for `ewallet` and `ledgerdb` but make a `vagrant` as non-superuser and allow it to connect without password (in `pg_hba.conf`). This way we can use the default `DATABASE_URL`. Thoughts?

I think the only inconsistency with prod-like environments is that the single user will be able to access cross-database while prod-like environments don't. But I don't think that would be an issue since `EWalletDB.Repo` and `LocalLedgerDB.Repo` couldn't make queries across the two databases anyway?

So a single `vagrant` non-superuser sounds good enough for the issue for now. I'll try that.